.changeset/witty-ligers-perform.md

```diff
@@ -0,0 +1,5 @@
+---
+'@mobile-reality/react-native-select-pro': minor
+---
+
+add custom select arrow icon prop
```

Comment:
you can add name of this prop here eg. "add custom select arrow icon prop `customSelectControlArrowIconSource`"